models/correlomatrix_detector.h

```diff
@@ -49,18 +49,18 @@ Device for measuring the covariance matrix from several inputs
 Description
 +++++++++++
 
-The correlomatrix_detector is a recording device. It is used to
+The ``correlomatrix_detector`` is a recording device. It is used to
 record spikes from several pools of spike inputs and calculates the
 covariance matrix of inter-spike intervals (raw auto and cross correlation)
-binned to bins of duration delta_tau. The histogram is only recorded for
+binned to bins of duration ``delta_tau``. The histogram is only recorded for
 non-negative time lags. The negative part can be obtained by the symmetry of
-the covariance matrix
- :math:` C(t) = C^T(-t)`.
-The result can be obtained via GetStatus under the key /count_covariance.
+the covariance matrix :math:`C(t) = C^T(-t)`.
+
+The result can be obtained via ``GetStatus`` under the key /count_covariance.
 In parallel it records a weighted histogram, where the connection weight are
 used to weight every count, which is available under the key /covariance.
-Both are matrices of size N_channels x N_channels, with each entry C_ij being
-a vector of size tau_max/delta_tau + 1 containing the (weighted) histogram
+Both are matrices of size :math:`N_channels x N_channels`, with each entry :math:`C_ij` being
```

Comment:
How does `:math:` handle the subscript? Could you check that `j` is also part of it? In LaTeX notation this would require curly braces `C_{ij}` to not end up as `C_{i} j`…

Reply:
Fixed these